action/envelope.go

```diff
@@ -256,3 +257,124 @@ func (elp *envelope) SetNonce(n uint64) { elp.nonce = n }
 
 // SetChainID sets the chainID value
 func (elp *envelope) SetChainID(chainID uint32) { elp.chainID = chainID }
+
+func (elp *envelope) loadProtoWithChainID(pbAct *iotextypes.ActionCore) error {
+	if pbAct == nil {
```

Comment:
create a private func to common code.

Reply:
if want to use common code, need to break down into 2 funcs.
this code will be reverted after 180 HF, so would like to keep it simple

Reply:
I also believe a private function for the shared code is needed.